src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,98 @@
+package duke;
```

Comment:
Perhaps you could start to create more specified packages for your files since you have already done it for command?